src/components/ux-alert/ux-alert.js

```diff
@@ -0,0 +1,9 @@
+Ractive.extend({
+	template: Ractive.defaults.templates['ux-alert'],
+	oninit: function ( options ) {
+		this.on( 'closeClicked', function () {
+			this.teardown();
+			return false;
```

Comment:
A.1) I think the click event should not trigger anything in my opinion. It just give the users more room to see their usage summary.
A.2) I assume so... teardown() will destroy the whole alert component, am I right?